libraries/lib-numeric-formats/NumericConverter.h

```diff
@@ -124,69 +46,59 @@ class NUMERIC_FORMATS_API NumericConverter /* not final */
    virtual void ControlsToValue();
 
 private:
-   void ParseFormatString(const TranslatableString & untranslatedFormat);
+   bool ParseFormatString(const TranslatableString & untranslatedFormat);
```

Comment:
It returns bool now, and not all false returns are ultimately checked (via UpdateFormatter or SetCustomFormat), but maybe it doesn't matter.

What mattered was to verify sufficient null checks of `mFormatter` (yes)